.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.3
+ruby-2.7.4
```

Comment:
Okay, one of my doubts was how to update stuff! Like Ruby version, Selenium version and etc.
At least for Ruby, is it simple as that?